api/v1alpha1/volumereplicationgroup_types.go

```diff
@@ -66,9 +66,19 @@ type VolumeReplicationGroupSpec struct {
 	// that needs to be replicated to the peer cluster.
 	PVCSelector metav1.LabelSelector `json:"pvcSelector"`
 
-	// ReplicationClass of all volumes in this replication group;
-	// this value is propagated to children VolumeReplication CRs
-	VolumeReplicationClass string `json:"volumeReplicationClass"`
+	// Important: Run "make" to regenerate code after modifying this file
+
+	// Label selector to identify the VolumeReplicationClass resources
+	// that are scanned to select an appropriate VolumeReplicationClass
+	// for the VolumeReplication resource.
+	//+optional
+	ReplicationClassSelector metav1.LabelSelector `json:"replicationClassSelector,omitempty"`
```

Comment:
ramen/config/samples/ramendr_v1alpha1_volumereplicationgroup.yaml needs to be updated